doc/source/examples/00-systemcoupling/heating_tank_fmu_fmu.py

```diff
@@ -0,0 +1,226 @@
+# data transfers, querying for setup errors, and modifying settings.
+#
+# Access the ``setup`` API
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+setup = syc.setup
+
+
+# %%
+# Add participants
+# ~~~~~~~~~~~~~~~~~
+# Use ``add_participant`` to create ``coupling_participant`` objects
+# representing the FMU participants, based on the previously defined
+# setup information.
+heater_part_name = setup.add_participant(input_file=fmu_file_heater)
+tank_part_name = setup.add_participant(input_file=fmu_file_tank)
+
+
+# %%
+# FMU settings
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Change FMU parameters by accessing ``fmu_parameter``
+
+# Change the "maximum heat output" settings
+setup.coupling_participant[heater_part_name].fmu_parameter["Real_2"].real_value = 10.0
+
+# %%
+# Change the "target temperature" settings
+setup.coupling_participant[heater_part_name].fmu_parameter["Real_3"].real_value = 350.0
+
+# %%
+# Change the "heat scale factor" settings
+setup.coupling_participant[heater_part_name].fmu_parameter["Real_4"].real_value = 2.0
+
+# %%
+# Create interfaces and data transfers
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create interfaces and data transfers by specifying participant variables
+# to map to one another. This consists of calling the appropriate commands
+# to create an interface and both temperature and heat flow data transfers.
+
+# Create a coupling interface for tank <-> heater (sensor, heat source)
+interface = setup.add_interface(
+    side_one_participant=tank_part_name, side_two_participant=heater_part_name
+)
+
+# Create data transfer for "temperature"
+temperatureDataTransfer = setup.add_data_transfer(
+    interface=interface,
+    target_side="Two",
+    source_variable="Real_0",
+    target_variable="Real_0",
+)
+
+# Create data transfer for "heatflow"
+heatFlowDataTransfer = setup.add_data_transfer(
+    interface=interface,
+    target_side="One",
+    source_variable="Real_1",
+    target_variable="Real_1",
+)
+
+# %%
+# Change the ``time_step_size`` setting.
+setup.solution_control.time_step_size = "1 [s]"
+
+# %%
+# Change the ``end_time`` setting.
+setup.solution_control.end_time = "50 [s]"
+
+# %%
+# Change the ``minimum_iterations`` and ``maximum_iterations`` settings.
+setup.solution_control.minimum_iterations = 1
+setup.solution_control.maximum_iterations = 5
+
+# %%
+# Set the ``option`` setting.
+setup.output_control.option = "StepInterval"
+
+# %%
+# Change the ``output_frequency`` frequency setting.
+setup.output_control.output_frequency = 2
+
+# %%
+# Change the ``generate_csv_chart_output`` setting to record the temperature
+# and heat rate values over time in a ``.csv`` file. This step is necessary
+# to chart the data after solving.
+setup.output_control.generate_csv_chart_output = True
+
+# %%
+# Run solution
+# ------------
+# The System Coupling server's ``stdout`` and ``stderr`` output is not shown
+# in PySystemCoupling by default. To see it, turn output streaming on.
```

Comment:
this comment isn't terribly helpful, imo, since we're not telling the user how to turn on the output streaming

Reply:
The following line of code turns on the output streaming. This is providing an explanation for the line of code.

That said, this level of detail was included in the original oscillating plate example as it was meant to be an introductory example. I am not sure that we need to keep repeating the same level of detail in all examples.